### PR TITLE
leagueoflegends.com

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/League_of_Legends.com.xml [hunk 1]
```diff
@@ -104,6 +103,7 @@
 	<target host="account.leagueoflegends.com" />
 	<target host="ddragon.leagueoflegends.com" />
 	<target host="cdn.leagueoflegends.com" />
+	<target host="status.leagueoflegends.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/League_of_Legends.com.xml]: ask.leagueoflegends.com
battlegrounds.leagueoflegends.com
oce.leagueoflegends.com
pbr.pbe.leagueoflegends.com
dominion.leagueoflegends.com

File: src/chrome/content/rules/League_of_Legends.com.xml [hunk 2]
```diff
@@ -47,7 +47,6 @@
 		- gameinfo.na ¹
 		- prized.na ²
 		- metrics ³
-		- status ³
```
Comment (J0WI) [inline src/chrome/content/rules/League_of_Legends.com.xml]: 500:
jp.leagueoflegends.com

HTTP redirect:
eune.leagueoflegends.com
ru.leagueoflegends.com
las.leagueoflegends.com
lan.leagueoflegends.com
br.leagueoflegends.com
tr.leagueoflegends.com

Mismatch:
boards.pbe.leagueoflegends.com

Timeout:
ulol.leagueoflegends.com

**Comment by galeksandrp** [comment]
Added, thanks!

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**